core/organization/service.go

```diff
@@ -184,6 +184,11 @@ func (s Service) AddMember(ctx context.Context, orgID, relationName string, prin
 	}); err != nil {
 		return err
 	}
+
+	audit.GetAuditor(ctx, orgID).Log(audit.OrgMemberCreatedEvent, audit.Target{
```

Comment:
This event is currently emitted in the org handler as well in the `AddOrganizationUsers` method. Can we remove it from there to avoid duplicate events?

Reply:
Can't find any other place, can you share direct git link?

Reply:
https://github.com/raystack/frontier/blob/e913c1c9dbc545951344bdccbd00c0bc776c4b1f/internal/api/v1beta1/org.go#L386

Reply:
I dont get it, why do you think it's not removed?

Reply:
<img width="1018" alt="Screenshot 2024-10-28 at 3 01 52 PM" src="https://github.com/user-attachments/assets/9316303e-479c-4995-a45f-2e555ad5ea5d">

Reply:
Missed that earlier while reviewing. Looks good.